src/xcode/ENA/ENA/Source/Workers/Logger.swift

```diff
@@ -16,27 +16,75 @@
 // under the License.
 
 import Foundation
+import SwiftyBeaver
 
 let appLogger = Logger()
 
-func log(message: String, level _: LogLevel = .info, file _: String = #file, line _: UInt = #line, function _: String = #function) {
-	NSLog("%@", message)
-}
-
-func logError(message: String, level _: LogLevel = .error, file _: String = #file, line _: UInt = #line, function _: String = #function) {
-	NSLog("%@", message)
-}
-
 class Logger {
-	func log(message _: String, level _: LogLevel = .info, file _: String, line _: UInt, function _: String) {}
+	
+	private let sb = SwiftyBeaver.self
+	
+	private let fileDest = FileDestination()
+	private let consoleDest = ConsoleDestination()
 
-	func getLoggedData() -> Data? {
-		Data()
+	private var logDirectory: URL? {
+		return try? FileManager.default.url(for: .cachesDirectory, in: .userDomainMask, appropriateFor: nil, create: true)
+			.appendingPathComponent("logs", isDirectory: true)
+	}
+	
+	private var logFileURL: URL? {
+		guard let logDir = logDirectory else {
+			return nil
+		}
+		
+		return logDir.appendingPathComponent("log.txt", isDirectory: false)
+	}
+	
+	init() {
+		setup()
+	}
+	
+	private func setup() {
+		fileDest.logFileURL = logFileURL
+	}
+	
+	func verbose(message: String, _ file: String = #file, _ line: Int = #line, _ function: String = #function) {
+		log(level: .verbose, message: message, file: file, line: line, function: function)
+	}
+	
+	func info(message: String, _ file: String = #file, _ line: Int = #line, _ function: String = #function) {
+		log(level: .info, message: message, file: file, line: line, function: function)
+	}
+	
+	func warning(message: String, _ file: String = #file, _ line: Int = #line, _ function: String = #function) {
+		log(level: .warning, message: message, file: file, line: line, function: function)
+	}
+	
+	func error(message: String, _ file: String = #file, _ line: Int = #line, _ function: String = #function) {
+		log(level: .error, message: message, file: file, line: line, function: function)
+	}
+	
+	private func log(level: SwiftyBeaver.Level, message: String, file: String, line: Int, function: String) {
+		#if DEBUG
```

Comment:
Could you please "reverse" the logic here to enable logs for all configurations that are not APP_STORE? We will add a new configuration "COMMUNITY" soon. 😊

Reply:
Just changed it. Please let me know if this is what you had in mind.

Reply:
Yes. Thank you!